src/shareSnapshot/share_rds_snapshot_across_accounts.py

```diff
@@ -0,0 +1,35 @@
+import os
+import boto3
+import constants
+import custom_exceptions
+
+def lambda_share_rds_snapshot_cross_account(event, context):
```

Comment:
Who is going to trigger this ?

Reply:
this will be triggered in the source aws account as part of the state machine

Reply:
ok